js/main-menu.js

```diff
@@ -104,7 +104,7 @@
     });
   }
 
-   /**
+  /**
```

Comment:
You really have no changes done in this file, please remove it from the PR

Reply:
Removed.